catalog/solutions/nri-cts/0.9.15/solution.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "nri-cts",
+  "name": "NRI-CTS",
```

Comment:
Please note that the value of this field will show up in the solution's card and will be searchable. You might want to consider using `"NRI Carbon Tracing System"` here.

Reply:
Isn't the `name` in `solution.json` supposed to be the name of our product? If so, NRI-CTS is fine.
I think the one corresponding to the search keywords is `summary` in `solution.json`, is it not?

Reply:
It is, indeed, the name of the product. I just wanted to make sure that you wanted it to be presented this way. Both the `name` and the `summary` are searchable. Thank you!